torch/_refs/__init__.py

```diff
@@ -213,7 +214,10 @@
 
 
 def _broadcast_shapes(*_shapes):
-    shapes = tuple(filter(lambda x: x is not None, _shapes))
+    shapes = tuple(
+        (x,) if isinstance(x, int) else x
```

Comment:
This is not the same thing though. We want to convert integer in `_shapes` to a tuple, because later we have 
```
for shape in shapes:
  assert isinstance(shape, Sequence)
  ... shape[idx] ...
```
It's actually a good catch and I forgot to add a counter example in the text. Will do.

Reply:
aha I see, thanks for the clarification